docs/pipelineruns.md

```diff
@@ -326,8 +326,6 @@ go through the complexity of checking each `Pipeline` and providing only the req
 
 #### Propagated Parameters
 
-**([alpha only](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features))**
```

Comment:
Sorry for running late on this review.
Wondering for this do we also need to add it to the [list of beta features](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#beta-features)?

Reply:
didn't see a list there and thought we'd create the list when we make the v1 release